KarhooUISDK/CommonUI/UIConstants.swift

```diff
@@ -207,6 +207,9 @@ public struct UIConstants {
 
         /// 12
         public static let xLarge: CGFloat = 12
+        
+        /// 16
+        public static let xxLarge: CGFloat = 16
```

Comment:
@BartlomiejSopalaKarhoo IIRC you introduced another entry for 20 and named it xxLarge as well. If that's the case, could you rename yours to xxxLarge to keep the ascending order?

Reply:
Sure, I renamed my property and copied your to my branch.